installation/aws/VmClarity.cfn

```diff
@@ -262,20 +262,22 @@ Resources:
                     VMCLARITY_AWS_SECURITY_GROUP_ID=${VmClarityScannerSecurityGroup}
                     # Name of the SSH KeyPair to use for Scanner instance launch
                     VMCLARITY_AWS_KEYPAIR_NAME=${KeyName}
-                    # The AMI image used for creating Scanner instance
-                    VMCLARITY_AWS_SCANNER_AMI_ID=${ScannerImageID}
-                    # The instance type used for Scanner instance
-                    VMCLARITY_AWS_SCANNER_INSTANCE_TYPE=${ScannerInstanceType}
+                    # ImageNameFilter contains the Name filter used for finding AMI to instantiate Scanner instance
+                    VMCLARITY_AWS_IMAGE_NAME_FILTER=${ScannerAMINameFilter}
+                    # Comma separated list of OwnerID(s)/OwnerAliases used as Owners filter for finding AMI
+                    # to instantiate Scanner instance.
+                    VMCLARITY_AWS_IMAGE_OWNERS=${ScannerAMIOwnersFilter}
+                    # Comma separated list of architecture:instance_type pairs used by the Provider to determine the instance type
+                    # for the Scanner instance based on the Targets architecture.
+                    VMCLARITY_AWS_INSTANCE_MAPPING=${ScannerInstanceTypeMapping}
+                    # Optional. Always use this architecture for Scanner instance. Make sure that there is a valid mapping defined in VMCLARITY_AWS_INSTANCE_MAPPING
+                    # for this architecture. 
```

Comment:
> Do we need this option? Could we just not specify a mapping for a specific arch, and the default behaviour if we don't have a mapping can be use the first arch specified in the mapping or something?

We can do this too. I just wanted to allow the user to explicitly set the arch they want to use instead of the implicit behaviour.

> It looks like the current behaviour is to fail if we have a target with an arch we don't understand, I'm not sure this is the best option as we should be able to at least try to scan with a scanner of a different arch and if the scanners fail then so be it.

I am not sure about falling back to a different arch is a good idea either. It is hard to predict the outcome of the scan as the scanner may fail, may report false positives, etc.
Anyways, will change this too to do a fallback.